packages/amplify-category-auth/package.json

```diff
@@ -14,17 +14,17 @@
   },
   "dependencies": {
     "amplify-category-function": "1.11.0",
-    "aws-sdk": "^2.475.0",
-    "chalk": "^2.4.1",
+    "aws-sdk": "^2.510.0",
+    "chalk": "^2.4.2",
     "chalk-pipe": "^2.0.0",
-    "fs-extra": "^7.0.0",
-    "inquirer": "6.3.1",
-    "lodash": "^4.17.10",
+    "fs-extra": "^8.1.0",
+    "inquirer": "^6.5.1",
```

Comment:
A few months back we had https://github.com/aws-amplify/amplify-cli/issues/1688 because of inquirer update. Could you make sure that has been fixed

Reply:
Pre 6.0 some downlevel stdio dependency caused all these problems, and I was using various categories during the smoke test and did not hit a hitch with it. If you've a specific test we should do then we can do it.

Reply:
From #1688 

> 
    Go to project and run amplify add api
    Choose Graphql
    Try to type name...freezes
    See error